googlemaps.py

```diff
@@ -0,0 +1,59 @@
+import requests
+
+
+def input_locations():
+    """
+    This allows the user to input the locations of the origin and destination and then puts these into the function to
+    get the distance
```

Comment:
Love the function documentations! I think that in production code the descriptions might be a bit more accurate. These are probably more helpful to us at the moment but we might want to change them before we submit. Technically this function doesn't put the origin and destination strings into the the get_distance function. It just returns origin and destination. A more accurate description would be:

This allows the user to input the locations of the origin and destination, assigns them to variables and returns these variables.

Reply:
Cheers, updated!